### DISCUSSION TITLE
How to deal with source term?

### DISCUSSION THREAD

**Discussion by babyQQQQ** [lululxvi/deepxde#1007] [Q&A]
Hello lu lu and deepxde teamers, I have a problem with helmholz equation, eg.https://deepxde.readthedocs.io/en/latest/demos/pinn_forward/helmholtz.2d.dirichlet.html. if the f is very large, for example 10000, the the training procedure is so slow, Could you please give me some help?

**Comment by lululxvi** [comment]
> f is very large, for example 10000

The value of f is large?
